docroot/themes/custom/uids_base/templates/block/block--inline-block--uiowa-card.html.twig

```diff
@@ -53,7 +53,7 @@
   'headline_level': content.field_uiowa_card_title[0]['#size'],
   'title_prefix': title_prefix,
   'title_suffix': title_suffix,
-  'visible_button': true,
+  'visible_button': field_uiowa_card_button_display,
```

Comment:
This appears to still be incorrect because the value of this field can be one of multiple strings which will always evaluate to true when used as a boolean.

Reply:
Okay... nevermind this comment, I see below that the value is being artificially set.